creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -95,8 +114,38 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
     return {
       error: {
         message: errorMsg
+      },
+      jobsToEnqueue: {}
+    }
+  }
+
+  if (syncMode === SYNC_MODES.MergePrimaryAndSecondary) {
+    if (!mergePrimaryAndSecondaryEnabled) {
+      logger.info(
+        `${logMsgString} || Sync mode is disabled - Will not issue sync request`
+      )
+      return {
+        error: {},
+        jobsToEnqueue: {}
       }
     }
+    /**
+     * For now, if primarySyncFromSecondary fails, we just log & error without any retries
+     * Eventually should make this more robust, but proceeding with caution
+     */
+
+    // Sync primary content from secondary and set secondary sync flag to forceResync before proceeding
+    const error = await primarySyncFromSecondary(secondaryEndpoint, userWallet)
```

Comment:
doesn't this take an obj of { wallet, secondary }

Reply:
😳 good find - fixed this bug and also added test coverage to ensure its being called with correct params

also these `issueSyncRequest` jobs are run in child processes per this code here https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/stateMachineManager/stateReconciliation/index.js#L148

why do you ask?

althoguh that question gave me an idea about maddog...

Reply:
just curius about the issueSync request. and thanks for fixing the params!

Reply:
coo